Automattic/wp-calypso
# G Suite: Post-checkout screen styles update

Related to #45123

## Proposed Changes

#### Style changes related to GSuite post-checkout screen (some style requirements in the issue need minor clarification and may be updated accordingly thereafter)

* Updating alert color from --color-warning to --studio-blue-10

* Adding external link icon for Google doc's link button

* Require some more clarification for other styles

## Testing Instructions



1. Purchase a plan with a custom domain
2. Add G Suite and check out
3. Note the post-checkout screen

**Before:**

![Before](https://github.com/Automattic/wp-calypso/assets/22623256/f7c79176-8e84-4477-b78b-a7198123571b)


**After:**

![After](https://github.com/Automattic/wp-calypso/assets/22623256/c62caed2-dfef-4ce0-a0d2-1817c3e126cc)

## Pre-merge Checklist
